.github/workflows/openvex.yaml

```diff
@@ -1,17 +1,28 @@
 on:
   workflow_dispatch:
-  push:
-    tags:
-      - 'v*.*.*'
+    inputs:
+      tag:
+        type: string
+        required: true
+
 jobs:
   vexctl:
     runs-on: ubuntu-latest
     steps:
+      - name: Set tag name
+        shell: bash
+        run: |
+          TAG=${{ github.event.inputs.tag }}
+          echo "TAG=$TAG" >> "$GITHUB_ENV"
       - name: Checkout code
         uses: actions/checkout@v2
-      - name: Set env
-        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
       - uses: openvex/generate-vex@31b415924ea0d72ed5f2640f1dee59dea6c2770b
         name: Run vexctl
         with:
-            product: pkg:golang/sigs.k8s.io/kueue@${{ env.RELEASE_VERSION }}
+          product: pkg:golang/sigs.k8s.io/kueue@${{ env.TAG }}
+          file: /tmp/kueve.vex.json
+      - name: Upload SBOM
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          gh release upload $TAG /tmp/kueve.vex.json
```

Comment:
@alculquicondor @tenzen-y  What do you guys think about this last step?

Reply:
seems fine. Do you have any concerns?